packages/documentation/copy/en/reference/Target Declaration.md

```diff
@@ -390,7 +390,7 @@ to point it to your preferred C++ compiler.
 ## docker
 <div class="lf-c lf-py lf-ts">
 
-This option takes a boolean argument. 
+This option takes a boolean argument (default is `false`). 
```

Comment:
Just out of curiosity: why is this the default? Is it costly to generate a docker file?

Reply:
I think because an LF program does not need a docker file to run by itself. The docker file is an addition to the default LF program

Reply:
That's true, but if it there's no cost, then why not just generate it always? Worst thing that could happen is that a programmer see it and realizes the functionality is there.

Reply:
I think most of the current use cases of LF programs do not involve containerization, and it would be cumbersome for the user to have to explicitly turn off the docker file generation. @Soroosh129 would you like to chime in on this?

Reply:
What would prompt a user to turn it off? This goes back to my earlier question: is there a cost?

Reply:
the cost is having more files in the generated folder. If we want to generate docker files regardless of the intended purpose of the program, this seems to suggest we should remove this target property and make the docker files part of the generated code.

Reply:
...or just have the default be `true`.